packages/taco/src/conditions/schemas/address-allowlist.ts

```diff
@@ -0,0 +1,14 @@
+import { EthAddressSchemaStrict } from '@nucypher/shared';
+import { z } from 'zod';
+
+export const addressAllowlistConditionSchema = z
+  .array(EthAddressSchemaStrict)
+  .min(1, 'At least one address must be provided')
+  .max(25, 'A maximum of 25 addresses is allowed')
+  .describe(
+    'List of wallet addresses allowed to decrypt. Addresses should be provided in checksummed form.',
```

Comment:
Let's not include "decrypt" in the description, since it can potentially be used as a signing condition as well.

```suggestion
    'List of allowed wallet addresses. Addresses should be provided in checksummed form.',
```